src/_codux/fake-data/fake-wix-api-context-provider.tsx

```diff
@@ -49,6 +49,16 @@ function getWixApi(settings?: Settings): WixAPI {
         getServiceAvailability: (lessonId: string) => {
             return Promise.resolve(createLessonAvailability(lessonId));
         },
+        initiateLogin: async () => {
+            alert('Initiate login');
+        },
+        fetchUserAuthData: async () => {
+            return Promise.resolve({ user: { member: { profile: { nickname: 'John Doe' } } } });
```

Comment:
Do you know if the user auth data is consistent? if so - please use types for clarity.
```suggestion
            const userAuthData = {
                user: {
                    member: {
                        profile: {
                            nickname: 'John Doe'
                        }
                    }
                }
            };
            return Promise.resolve(userAuthData);
```

Reply:
added type to user auth data